build/lib/android/androidsdk.js

```diff
@@ -3,7 +3,7 @@
 const os = require('os');
 const fs = require('fs-extra');
 const path = require('path');
-const DEFAULT_API_LEVEL = 28; // FIXME: Use android's package.json compileSDKVersion value
+const DEFAULT_API_LEVEL = require(path.join(__dirname, '../../../android/package.json')).compileSDKVersion;
```

Comment:
<!--
  0 failure: 
  1 warning:  build/lib/android...
  
  
  DangerID: danger-id-Danger;
  File: build/lib/android/androidsdk.js;
  Line: 6;
-->

- :warning: build/lib/android/androidsdk.js line 6 – Found non-literal argument in require (security/detect-non-literal-require)